pkg/apis/kops/v1alpha2/componentconfig.go

```diff
@@ -179,6 +179,8 @@ type KubeletConfigSpec struct {
 	AuthenticationTokenWebhook *bool `json:"authenticationTokenWebhook,omitempty" flag:"authentication-token-webhook"`
 	// AuthenticationTokenWebhook sets the duration to cache responses from the webhook token authenticator. Default is 2m. (default 2m0s)
 	AuthenticationTokenWebhookCacheTTL *metav1.Duration `json:"authenticationTokenWebhookCacheTtl,omitempty" flag:"authentication-token-webhook-cache-ttl"`
+	// RegistryQPS if > 0, limit registry pull QPS to this value.  If 0, unlimited. (default 5)
+	RegistryQPS *int32 `json:"registryQPS,omitempty" flag:"registry-qps"`
```

Comment:
Could we make this ` RegistryPullQPS *int32 `json:"registryPullQPS,omitempty" flag:"registry-qps"``

That is what the kubelet component config is going to be https://github.com/kubernetes/kubernetes/blob/d5a3db003916b1d33b503ccd2e4897e094d8af77/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L210